datalab/datalab_session/analysis/raw_data.py

```diff
@@ -0,0 +1,31 @@
+import numpy as np
+from PIL import Image
+from datalab.datalab_session.s3_utils import get_fits
+from datalab.datalab_session.file_utils import get_hdu
+from fits2image.scaling import extract_samples, calc_zscale_min_max
+
+def raw_data(input: dict):
+    fits_path = get_fits(input['basename'], input.get('source', 'archive'))
+
+    sci_hdu = get_hdu(fits_path, 'SCI')
+    image_data = sci_hdu.data
+    
+    # Compute the fits2image autoscale params to send with the image
+    samples = extract_samples(image_data, sci_hdu.header, 2000)
+    median = np.median(samples)
+    _, zmax, _ = calc_zscale_min_max(samples, contrast=0.1, iterations=1)
+
+    # resize the image to max. 500 pixels on an axis
```

Comment:
500 or 800?

Reply:
yeah were requesting 500 from the frontend, but i guess i made the default 800. confusing